### ISSUE TITLE
Crash on analyze video

### ISSUE THREAD

**Issue by LWitter** [DeepLabCut/DeepLabCut#2026]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I have unexplained crashes on analyze video.
I try to analyze movies from open field data where four cages of mice were recorded simultaneously via 1 camera. I have split movies into four separate clips (cropping 4x each movie) and analyzed these with DLC. Worked great.
Now, when I try to use analyze videos ipython crashes, even when using the exact same movies that previously were used for labeling. The crash is without a command-line error. Just a windows-style crash with a popup stating 'python has stopped working'.

Now the weird part: When I try to analyze a non-cropped movie, everything works fine. Which led me to believe something changed in the codec making DLC or something under the hood not properly recognizing the movie. I tried every coded I could think using VLC. Nothing worked. So, I switched to ffmpeg to split movies and code them. Nothing worked. Then I realized DLC has a crop function! Didn't work either, not via the config file, nor via the GUI. Now, again I tried analyzing the non-cropped movie. Worked.
Very confusing behavior.
I also tried not filtering the points, not exporting the CSV file, etc. Didn't work....

What could be going on here? Does DLC maybe want a larger video (in terms of X*Y pixels)? Should I rebuild the project and try again? Did anyone experience this before?

### Operating System

Windows 7 (required by my institute on managed R&D computers)

### DeepLabCut version

dlc version 2.2.0.3

### DeepLabCut mode

single animal

### Device type

GPU: RTX2080TI

Following should be irrelevant I think?
CPU: Xeon W2123
Memory: 128Gb

### Steps To Reproduce

(deeplabcut) C:\Users\lwitter2>pip list
Package                       Version
----------------------------- -------------------
absl-py                       1.0.0
anipose                       1.0.1
aniposelib                    0.4.3
apptools                      5.1.0
argon2-cffi                   21.1.0
astunparse                    1.6.3
async-generator               1.10
attrs                         21.2.0
backcall                      0.2.0
backports.functools-lru-cache 1.6.4
bleach                        4.1.0
Brotli                        1.0.9
cachetools                    4.2.4
certifi                       2021.10.8
cffi                          1.15.0
charset-normalizer            2.0.9
click                         8.0.4
colorama                      0.4.4
configobj                     5.0.6
cycler                        0.11.0
debugpy                       1.5.1
decorator                     5.1.1
deeplabcut                    2.2.0.3
defusedxml                    0.7.1
entrypoints                   0.3
filterpy                      1.4.5
Flask                         2.0.3
Flask-Compress                1.11
flask-ipban                   1.1.5
flatbuffers                   2.0
fonttools                     4.28.3
gast                          0.4.0
google-auth                   2.3.3
google-auth-oauthlib          0.4.6
google-pasta                  0.2.0
grpcio                        1.42.0
h5py                          3.6.0
idna                          3.3
imageio                       2.13.3
imageio-ffmpeg                0.4.5
imgaug                        0.4.0
importlib-metadata            4.8.2
importlib-resources           5.4.0
ipykernel                     6.5.1
ipython                       7.30.1
ipython-genutils              0.2.0
ipywidgets                    7.6.5
itsdangerous                  2.1.1
jedi                          0.18.1
Jinja2                        3.0.3
joblib                        1.1.0
jsonschema                    4.2.1
jupyter                       1.0.0
jupyter-client                7.1.0
jupyter-console               6.4.0
jupyter-core                  4.9.1
jupyterlab-pygments           0.1.2
jupyterlab-widgets            1.0.2
keras                         2.8.0
Keras-Preprocessing           1.1.2
kiwisolver                    1.3.2
libclang                      12.0.0
llvmlite                      0.37.0
Markdown                      3.3.6
MarkupSafe                    2.0.1
matplotlib                    3.5.0
matplotlib-inline             0.1.3
mistune                       0.8.4
mkl-service                   2.4.0
moviepy                       1.0.3
msgpack                       1.0.3
msgpack-numpy                 0.4.7.1
nb-conda                      2.2.1
nb-conda-kernels              2.3.1
nbclient                      0.5.9
nbconvert                     6.3.0
nbformat                      5.1.3
nest-asyncio                  1.5.4
networkx                      2.6.3
notebook                      6.4.6
numba                         0.54.1
numexpr                       2.8.1
numpy                         1.20.0
oauthlib                      3.1.1
opencv-contrib-python         4.5.5.64
opencv-python                 4.5.4.60
opencv-python-headless        4.5.4.60
opt-einsum                    3.3.0
packaging                     21.3
pandas                        1.3.4
pandocfilters                 1.5.0
parso                         0.8.3
patsy                         0.5.2
pickleshare                   0.7.5
Pillow                        8.4.0
pip                           21.3.1
proglog                       0.1.9
prometheus-client             0.12.0
prompt-toolkit                3.0.24
protobuf                      3.19.1
psutil                        5.8.0
pyasn1                        0.4.8
pyasn1-modules                0.2.8
pycparser                     2.21
pyface                        7.4.1
Pygments                      2.10.0
pyparsing                     3.0.6
PyQt5                         5.12.3
PyQt5_sip                     4.19.18
PyQtChart                     5.12
PyQtWebEngine                 5.12.1
pyrsistent                    0.18.0
python-dateutil               2.8.2
pytz                          2021.3
PyWavelets                    1.2.0
pywin32                       302
pywinpty                      1.1.6
PyYAML                        6.0
pyzmq                         22.3.0
qtconsole                     5.2.1
QtPy                          1.11.3
requests                      2.26.0
requests-oauthlib             1.3.0
rsa                           4.8
ruamel.yaml                   0.17.17
ruamel.yaml.clib              0.2.6
scikit-image                  0.18.1
scikit-learn                  1.0.1
scikit-video                  1.1.11
scipy                         1.7.3
Send2Trash                    1.8.0
setuptools                    59.4.0
setuptools-scm                6.3.2
Shapely                       1.8.0
six                           1.16.0
statsmodels                   0.13.1
tables                        3.7.0
tabulate                      0.8.9
tensorboard                   2.8.0
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.0
tensorflow                    2.8.0
tensorflow-estimator          2.7.0
tensorflow-io-gcs-filesystem  0.24.0
tensorpack                    0.11
termcolor                     1.1.0
terminado                     0.12.1
testpath                      0.5.0
tf-estimator-nightly          2.8.0.dev2021122109
tf-slim                       1.1.0
threadpoolctl                 3.0.0
tifffile                      2021.11.2
toml                          0.10.2
tomli                         1.2.2
tornado                       6.1
tqdm                          4.62.3
traitlets                     5.1.1
traits                        6.3.2
traitsui                      7.3.1
typing_extensions             4.0.1
urllib3                       1.26.7
wcwidth                       0.2.5
webencodings                  0.5.1
Werkzeug                      2.0.2
wheel                         0.37.0
widgetsnbextension            3.5.2
wrapt                         1.13.3
wxPython                      4.1.1
zipp                          3.6.0

### Relevant log output

```shell
In [3]: deeplabcut.launch_dlc()
Using snapshot-950000 for model D:\DLC\Open_Field_Michael\OpenFieldMichael-LW-20
22-07-15\dlc-models\iteration-0\OpenFieldMichaelJul15-trainset95shuffle1
C:\ProgramData\Anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\ker
as\engine\base_layer_v1.py:1694: UserWarning: `layer.apply` is deprecated and wi
ll be removed in a future version. Please use `layer.__call__` method instead.
  warnings.warn('`layer.apply` is deprecated and '
C:\ProgramData\Anaconda3\envs\deeplabcut\lib\site-packages\tf_slim\layers\layers
.py:684: UserWarning: `layer.apply` is deprecated and will be removed in a futur
e version. Please use `layer.__call__` method instead.
  outputs = layer.apply(inputs, training=is_training)
2022-10-27 10:12:50.276290: I tensorflow/core/platform/cpu_feature_guard.cc:151]
 This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (on
eDNN) to use the following CPU instructions in performance-critical operations:
 AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate comp
iler flags.
2022-10-27 10:12:51.136290: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1
525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 9526 MB me
mory:  -> device: 0, name: NVIDIA GeForce RTX 2080 Ti, pci bus id: 0000:65:00.0,
 compute capability: 7.5
Starting to analyze %  D:\DLC\Open_Field_Michael\OpenFieldMichael-LW-2022-07-15\
videos\Trial1cropped.mpg
Loading  D:\DLC\Open_Field_Michael\OpenFieldMichael-LW-2022-07-15\videos\Trial1c
ropped.mpg
Duration of video [s]:  655.92 , recorded with  25.0 fps!
Overall # of frames:  16398  found with (before cropping) frame dimensions:  202
 204
Starting to extract posture
  0%|                                                | 0/16398 [00:00<?, ?it/s]2
022-10-27 10:12:53.749290: I tensorflow/stream_executor/cuda/cuda_dnn.cc:368] Lo
aded cuDNN version 8101

(deeplabcut) C:\Users\lwitter2>
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by neslihanedes** [comment]
Hi @LWitter,

I had a similar problem before. Encoding the videos with ffmpeg as below solved my problem:

`ffmpeg -i "video.mp4" -c:v libx264 -crf 23 -c:a copy "video_copy.mp4"`

**Comment by LWitter** [comment]
Sorry for the slow reply, I couldn't test this earlier.
Today I tried recoding the movies as suggested. Unfortunately, still the exact same issue. Again, tested the non-cropped 4-view videos, no issue. So, it is still weird.
Might it not be related to the encoder at all in some way then? Might it be an issue of movies size? Has anyone ever experienced that? (Or am I talking nonsense?)

**Comment by jeylau** [comment]
@LWitter, would you like to share one such cropped video (a link via email for example) so I could see what's wrong?

**Comment by jeylau** [comment]
Closing, as this had been addressed by email with @AlexEMG ☺️
